3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/EnrichmentConfigsSpec.scala

```diff
@@ -122,7 +125,11 @@ class EnrichmentConfigsSpec extends Specification with ValidationMatchers {
             "mktSource": ["utm_source", "source"],
             "mktTerm": ["utm_term"],
             "mktContent": [],
-            "mktCampaign": ["utm _ campaign", "CID", "legacy-campaign!?-`@#$%^&*()=\\][}{/.,<>~|"]
+            "mktCampaign": ["utm _ campaign", "CID", "legacy-campaign!?-`@#$%^&*()=\\][}{/.,<>~|"],
+            "mktClickId": {
+              "customclid": "Custom",
+              "gclid": "Override"
```

Comment:
This is testing the ability to override the built in mappings (like "gclid" -> "Google") with custom ones (like "gclid -> "Override").